tests/pre-command.bats

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bats
+
+setup() {
+  load "$BATS_PLUGIN_PATH/load.bash"
+}
+
+@test "Exports credentials" {
+    export BUILDKITE_PLUGIN_GCP_WORKLOAD_IDENTITY_FEDERATION_AUDIENCE="//iam.googleapis.com/projects/123456789/locations/global/workloadIdentityPools/buildkite-example-pipeline/providers/buildkite"
+    export BUILDKITE_PLUGIN_GCP_WORKLOAD_IDENTITY_FEDERATION_SERVICE_ACCOUNT="buildkite-example-pipeline@oidc-project.iam.gserviceaccount.com"
+
+    TMPREGEX="Wrote credentials to (/tmp/tmp\.[a-zA-Z0-9]+)"
```

Comment:
a simpler way to avoid having to use a regex on the output would be to stub `mktmp` to return a specific value (remember to remove the file afterwards)